pre-commit

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Could we split the commit hook into a different pull request?

Reply:
Done!